### PR TITLE
Added Answer entity

### PR DESCRIPTION
**Pull request by tarasgl**

### CODE REVIEW

File: src/main/java/com/softserve/academy/studhub/entity/Answer.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@EqualsAndHashCode
+@Entity
+public class Answer {
```
Comment (volodymyroseredchuk) [inline src/main/java/com/softserve/academy/studhub/entity/Answer.java]: Add @Table annotation

File: src/main/java/com/softserve/academy/studhub/entity/Answer.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    @Column(name = "id")
+    private Integer id;
+
+    @EqualsAndHashCode.Exclude
+    @Column(name="body", columnDefinition = "text")
+    private String body;
+
+    @EqualsAndHashCode.Exclude
+    @Column(name = "approved")
+    private boolean approved;
```
Comment (Jarvizzik) [inline src/main/java/com/softserve/academy/studhub/entity/Answer.java]: boolean -> Boolean

File: src/main/java/com/softserve/academy/studhub/entity/Answer.java (hunk 1 above)
Comment (tarasgl) [reply src/main/java/com/softserve/academy/studhub/entity/Answer.java]: fixed

File: src/main/java/com/softserve/academy/studhub/entity/Answer.java (hunk 2 above)
Comment (tarasgl) [reply src/main/java/com/softserve/academy/studhub/entity/Answer.java]: fixed

**Pull request merged**